docs/cugraph/source/graph_support/PyG_support.md

```diff
@@ -0,0 +1,24 @@
+# cugraph_pyg
```

Comment:
This document is out of date.  Usage of cugraph-pyg is now significantly different from what is shown in the included example.  In addition, cugraph-service is no longer used by cugraph-pyg; cugraph-pyg now depends exclusively on cugraph.

Reply:
removed that specific info in that page. I will replace it with better text and an example when I update the page in 23.06